src/_playground/Routes.js

```diff
@@ -327,10 +341,12 @@ export class Routes extends Component {
             );
         });
 
+        const showNav = this.state.showSideNav ? '' : 'sidebar-hidden';
```

Comment:
This `sidebar-hidden` class name is potentially misleading.  Since the CSS media queries and the state in this component aren't always in sync, the `sidebar-hidden` class actually appears when the side bar _isn't_ hidden.  I get what you're going for here since you need a class for the purposes of a selector, but maybe a different name.  Maybe something like `sidebar-hidable`?  This would indicate it can be hidden if the CSS media query dictates.

Reply:
i've changed it to `sidebar-visibility`, does that work?